Editor/EditorCore/ProBuilderEditor.cs

```diff
@@ -584,7 +584,19 @@ void OnSceneGUI(SceneView sceneView)
 
         internal void HandleMouseEvent(SceneView sceneView, int controlID)
         {
-            if(m_CurrentEvent.type == EventType.MouseDown && m_CurrentEvent.keyCode == KeyCode.Mouse0 && HandleUtility.nearestControl == controlID)
+            if (Tools.viewToolActive || Tools.current == Tool.View || (Event.current.modifiers & EventModifiers.Alt) == EventModifiers.Alt)
+                return;
+
+#if UNITY_2023_1_OR_NEWER
+            bool isLeftMouseClick = m_CurrentEvent.type == EventType.MouseDown && m_CurrentEvent.keyCode == KeyCode.Mouse0 && HandleUtility.nearestControl == controlID;
+#elif UNITY_2022_3
+            KeyCode keyCode = m_CurrentEvent.isMouse ? m_CurrentEvent.button + KeyCode.Mouse0 : m_CurrentEvent.keyCode;
+            bool isLeftMouseClick = m_CurrentEvent.type == EventType.MouseDown && keyCode == KeyCode.Mouse0 && HandleUtility.nearestControl == controlID;
+#else
+            bool isLeftMouseClick = m_CurrentEvent.type == EventType.MouseDown && HandleUtility.nearestControl == controlID;
+#endif
+
+            if (isLeftMouseClick)
```

Comment:
Check with @claudiaantoun on this - I think her PR simplified this logic

Reply:
Sorry, that was already modified but not pushed :D